File: PHP/8 kyu/Switch it Up!.php
<?php
/*
 

DESCRIPTION  

When provided with a number between 0-9, return it in words.

Input :: 1

Output :: "One".

If your language supports it, try using a <a href="https://en.wikipedia.org/wiki/Switch_statement">switch statement</a>.


*/

function switch_it_up($number)
{
  switch ($number)
  {
  case 0:
  return "Zero";
  case 1:
  return "One";
  case 2:
  return "Two";
  case 3:
  return "Three";
  case 4:
  return "Four";
  case 5:
  return "Five";
  case 6:
  return "Six";
  case 7:
  return "Seven";
  case 8:
  return "Eight";
  case 9:
  return "Nine";
}
}

File: Python/8 kyu/Switch it Up!.py

 """ 
When provided with a number between 0-9, return it in words.

Input :: 1

Output :: "One".

If your language supports it, try using a <a href="https://en.wikipedia.org/wiki/Switch_statement">switch statement</a>.
 """ 

def switch_it_up(number):
    numb={
    0:"Zero",
    1:"One",
    2:"Two",
    3:"Three",
    4:"Four",
    5:"Five",
    6:"Six",
    7:"Seven",
    8:"Eight",
    9:"Nine"
    }
    return numb[number]